spark/dl/src/main/scala/com/intel/analytics/bigdl/transform/vision/image/util/BboxUtil.scala

```diff
@@ -477,4 +477,108 @@ object BboxUtil {
     }
     indices
   }
+
+  private def decodeSignalBoxWithWeight(encodeBox: Tensor[Float], bbox: Tensor[Float],
+             weight: Array[Float], decodeBox: Tensor[Float]): Unit = {
+    require(bbox.nDimension() == 1 && encodeBox.nDimension() == 1 && decodeBox.dim() == 1,
+    s"Only support decode single bbox, but get ${bbox.nDimension()}, ${encodeBox.nDimension()}")
```

Comment:
why missing decodebox dimension in output log ?

Reply:
ok, changed